src/meta/meta_service.cpp

```diff
@@ -52,6 +52,14 @@
 namespace dsn {
 namespace replication {
 
+DSN_DEFINE_uint64("meta_server",
+                  min_live_node_count_for_unfreeze,
+                  3,
+                  "minimum live node count without which the state is freezed");
+DSN_TAG_VARIABLE(min_live_node_count_for_unfreeze, FT_MUTABLE);
+DSN_DEFINE_validator(min_live_node_count_for_unfreeze,
+                     [](uint64_t min_live_node_count) -> bool { return min_live_node_count >= 0; });
```

Comment:
why did you change the validator from `>` to `>=`?
And the type of min_live_node_count_for_unfreeze is uint64_t, so if there is no need to add a validator for `>=`

Reply:
Actually in the unit test it is set 0:
```
svc->_meta_opts.min_live_node_count_for_unfreeze = 0;
```

Since it was previously set as a property of `meta_options`, zero is allowed; however, once it's not allowed zero for `FLAGS_min_live_node_count_for_unfreeze`, its validation will fail. So I changed it to ">=". Can we set it directly by `FLAGS_min_live_node_count_for_unfreeze  = 0` around the validator with it still kept as ">"?

I reserved the validator of `min_live_node_count_for_unfreeze`  for the possible update future

Reply:
I think it's ok to do like you said